scripts/scripts.js

```diff
@@ -105,6 +109,11 @@ async function loadLazy(doc) {
   loadFooter(doc.querySelector('footer'));
 
   loadCSS(`${window.hlx.codeBasePath}/styles/lazy-styles.css`);
+
+  loadCSS(`${window.hlx.codeBasePath}/styles/fonts.css`, () => {
+    if (!window.location.hostname.includes('localhost')) sessionStorage.setItem('fonts-loaded', 'true');
```

Comment:
Why `sessionStorage` and not `localStorage` ? If I frequently come back on the site on mobile (and close my browser app), I'll have to wait before the font is loaded while the font is cached already.

Reply:
Loading fonts.css a 2nd time on desktop... again, this probably works but not necessarily nice. We could use a global flag to track if we have loaded fonts.css or not. And also set the sessionStorage property in the eager phase too.

Reply:
`localStorage` is permanent, by default we cache for 2h for items coming from the code bus which seems much more of a `session` scope...

Reply:
i think `loadCSS()` would not add the `fonts.css` if it is already present, and if it did, the browser wouldn't load it twice either, so i think this is pretty safe.

Reply:
`sessionStorage` may throw an exception if quota has been exceeded or user disabled web storage. You may want to wrap in a try catch.

Reply:
good point. i have definitely never done that in the past, but i am not opposed to it.